depth_first_traversal/DepthFirstTraversal.py

```diff
@@ -0,0 +1,25 @@
+def depthFirstTraversal(graph, node, visited):
```

Comment:
Great code. Thanks for that!

Would you like to add the iterative approach too?

Reply:
Sure, I'll add it in the same file